workshop/exercise-3/README.md

```diff
@@ -1,79 +1,64 @@
+## Deploy our First Application to Knative using the Knative Client (kn)
+A relatively new project, the knative client aims to make interacting with Knative a seamless experience for developers. We'll try deploying an application using `kn`, then we'll deploy the same application using `kubectl` and `.yaml` files for additional control. The application for this lab is a simple node.js with express app which returns the first n numbers of the Fibonacci sequence. To use the app, deploy it, and simply make a GET request to the `/` endpoint with a number as the parameter. 
 
-Knative is currently built on top of both Kubernetes and Istio.
-If you want to learn more about Kubernetes or Istio, you can check out the
-labs [Kube101](https://github.com/IBM/kube101/tree/master/workshop) and
-[Istio101](https://github.com/IBM/istio101/tree/master/workshop).
-When you install Knative on IKS, it will install Istio for you
-automatically.
+# Our First Knative Service
+Knative Serving supports deploying and serving of serverless applications and functions. Those applications and functions will automatically scale up, and then back down to zero. In this exercise, we'll use the Knative Serving component to deploy our first application from a container image hosted on dockerhub. 
 
-### Install Knative
+Knative Services automatically manage the whole lifecycle for your workload. A service represents your app on Knative. Services control the creation of other objects to ensure that your app has a route, a configuration, and a new revision for each update of the service.
 
-1. To install Knative, first make sure you have the latest Kubernetes
-   plugin:
+![](https://github.com/knative/serving/raw/master/docs/spec/images/object_model.png)
+
+# Deploy our Application to Knative using kn
+We've already created an image on dockerhub that contains the first version of our Fibonacci application. If we call the `/` endpoint, and pass in a `number` parameter, we should get the first `n` numbers of the Fibonacci sequence.
+
+1. Deploy the application:
 
     ```
-    ibmcloud plugin update
+    kn service create --image docker.io/ibmcom/fib-knative fib-knative
     ```
 
-2. Next ask for Knative to be installed:
+2. Watch the pods initializing as our application gets deployed and starts up:
 
     ```
-	ibmcloud ks cluster-addon-enable knative --cluster $MYCLUSTER
-	```
+    kubectl get pods --watch
+    ```
+
+    Note: To exit the watch, use `ctrl + c`.
 
-3. The install process may take a minute or two. To know when it's done you
-   can run two commands - first see if the Istio and Knative namespaces
-   are there:
+3. Let's try out our new application! First, let's get the domain name that Knative assigned to the Route for the Service we just deployed. Run the following command, and note the value for `domain`. IBM Cloud Kubernetes Service sets the default domain name for Knative to match the domain name of your IBM Cloud Kubernetes Service cluster.
```

Comment:
I think it would be best to avoid mentioning Routes and just use `kn service get fib-knative` since that's the tool we're pushing them towards in this part of the demo. In fact they can use that to know when the service is ready too - instead of checking the pods.  In fact checking the pods is risky since they could vanish if they're slow

Reply:
updated.